### DISCUSSION TITLE
Column Span with Breakpoint Prefix

### DISCUSSION THREAD

**Discussion by bloewen404** [tailwindlabs/tailwindcss#14021] [Help]
I am trying to make a column inside a grid span 3 columns instead of 1. The library I'm using doesn't provide customization of the parent element, just the ability to add classes to the child columns.

The classes look like:
```html
<div class="grid gap-6 md:grid-cols-3">
  <div class="md:col-span-3">
     ...
  </div>
</div>
```

However this doesn't work. If I remove the "md:" from both classes in my browser tools I get the expected behaviour. I cannot remove or change the "md:grid-cols-3" on the parent, only the child.

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.

Otherwise, consider providing a project that reproduces the unexpected behavior.
